crates/fs/Cargo.toml

```diff
@@ -46,6 +46,7 @@ env_logger = "0.9"
 proptest = "1.0"
 test-log = "0.2"
 test-strategy = "0.2"
+criterion = { version = "0.4.0", features = ["async_std"] }
```

Comment:
minor: just 0.4, let the patch move.

Reply:
@zeeshanlakhani Also. Curious. What is benefit of not providing the patch?

Reply:
@appcypher typically libs shouldn't break compat at the patch level, so when building you'll automatically get the newest minor in your lock file.

Reply:
There are obvs bad deps/libs out there tho :).